packages/OmegaPrint-Tests.package/TestUI.class/README.md

```diff
@@ -0,0 +1 @@
+The TestUI class holds all tests for the UI namely the button a codeholder. It tests its existence and the functionality.
```

Comment:
button and codeholder. It tests the existence of the button and the functionality